app/views/CreditsSummary.scala.html

```diff
@@ -111,8 +115,8 @@
 
 @moneyInYourAccount(isAgent: Boolean) = {
     @maybeBalanceDetails.map { balanceDetails =>
-        @balanceDetails.availableCredit.map { availableCredit =>
-            @h2(msg = "credits.money-in-your-account-section.name", classes = "govuk-heading-s", optId = Some("h2-money-in-your-account"))
+        @balanceDetails.availableCredit.filter(_ > 0.00).map { availableCredit =>
```

Comment:
we have a full control of what type we are passing to this view, why not to do this transformation on the controller level at least and pass appropriate type in the view and make it "lighter" ?

Reply:
To be honest I can add it to the CreditService.maybeBalanceDetails() ?

Reply:
Hi, I've moved logic to the controller which simplified a view a bit. thanks